game_of_life_test_review.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+import game_of_life
+import test
+
+
+def test_newCellStateTrue():
+    assert game_of_life.new_cell_state(game_of_life.first_gen, 1, 3) == "."
```

Comment:
Remind, plz, what we discussed on the lection about unit test - a test should give answers, don't raise them

My questions to this test:
1. what is first_gen? Suggestion: put small test field directly in the test
2. what is `1, 3`? Suggestion: in Python you have named arguments - `game_of_life.new_cell_state(game_of_life.first_gen, cell_x=1, cell_y=3)`
3. what is `.`? Actually, it's an input/output detail - it might be better to use boolean value here and convert it into character in some `output` method

But overall it's awesome that you have this test!